File: CS/FilterDeserializer/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace FilterDeserializer {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }
        string layoutFileName = "test.xml";
        private void Form1_Load(object sender, EventArgs e) {
            DataTable tbl = new DataTable();
            tbl.Columns.Add("ID", typeof(int));
            tbl.Columns.Add("Name", typeof(string));
            tbl.Columns.Add("Date", typeof(DateTime));
            tbl.Columns.Add("Payment", typeof(decimal));
            for(int i = 1; i < 30; i++)
                tbl.Rows.Add(i, string.Format("Item{0}", i % 7), DateTime.Now.AddDays(-60).AddDays(i), i * 3.3);
            gridControl1.DataSource = tbl;
        }

        private void btnSaveLayout_Click(object sender, EventArgs e) {
            gridView1.SaveLayoutToXml(layoutFileName, DevExpress.Utils.OptionsLayoutGrid.FullLayout);
        }

        private void btnRestoreLayout_Click(object sender, EventArgs e) {
            if(System.IO.File.Exists(layoutFileName))
                FilterDeserializer.LoadFilter(gridView1, layoutFileName);
        }
    }
}

File: VB/FilterDeserializer/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace FilterDeserializer
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub
		Private layoutFileName As String = "test.xml"
		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Dim tbl As New DataTable()
			tbl.Columns.Add("ID", GetType(Integer))
			tbl.Columns.Add("Name", GetType(String))
			tbl.Columns.Add("Date", GetType(DateTime))
			tbl.Columns.Add("Payment", GetType(Decimal))
			For i As Integer = 1 To 29
				tbl.Rows.Add(i, String.Format("Item{0}", i Mod 7), DateTime.Now.AddDays(-60).AddDays(i), i * 3.3)
			Next i
			gridControl1.DataSource = tbl
		End Sub

		Private Sub btnSaveLayout_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnSaveLayout.Click
			gridView1.SaveLayoutToXml(layoutFileName, DevExpress.Utils.OptionsLayoutGrid.FullLayout)
		End Sub

		Private Sub btnRestoreLayout_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRestoreLayout.Click
			If System.IO.File.Exists(layoutFileName) Then
				FilterDeserializer.LoadFilter(gridView1, layoutFileName)
			End If
		End Sub
	End Class
End Namespace